staging/src/k8s.io/kubectl/pkg/generate/versioned/run_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"reflect"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
why is this necessary?

Reply:
vscode added it, but I kept it because I thought maybe golang standard wants us using that specifically when multiple paths have same pkg name.